### DISCUSSION TITLE
example of ObjectDetectionTask

### DISCUSSION THREAD

**Discussion by roybenhayun** [torchgeo/torchgeo#1655] [Q&A]
Hi,
can anyone please point to a base example of training with [ObjectDetectionTask](https://torchgeo.readthedocs.io/en/stable/api/trainers.html#torchgeo.trainers.ObjectDetectionTask)? 

a good reference example, e.g., in a notebook like[ training-example](https://github.com/microsoft/satellite-imagery-labeling-tool/blob/main/example/training-example.ipynb), is very needed when doing first steps. 

thanks in advance!

**Comment by adamjstewart** [comment]
https://gist.github.com/calebrob6/d32673997661a75d1a5e2e5c0afbe68a
